### DISCUSSION TITLE
Flush hang when loading partition into memory: ["DataCoord receive get flush state request"] [[flush state] unflushed segment  ids]

### DISCUSSION THREAD

**Discussion by NehaTamore-Draup** [milvus-io/milvus#23319] [Q&A and General discussion]
- Milvus version: 
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2): v2.1.2
- OS(Ubuntu or CentOS): Ubuntu
- CPU/Memory: 32GB
- GPU: 
- Others:

**Comment by yhmo** [comment]
Please create an issue in the [issues](https://github.com/milvus-io/milvus/issues) panel.
And provide more information including:
- logs of the cluster
- problem detail description
